packages/swapper/src/swappers/PortalsSwapper/constants.ts

```diff
@@ -0,0 +1,21 @@
+import type { ChainId } from '@shapeshiftoss/caip'
+import { KnownChainIds } from '@shapeshiftoss/types'
+
+import type { SupportedChainIds } from '../../types'
+import { PortalsSupportedChainIds } from './types'
+
+export const PORTALS_SUPPORTED_CHAIN_IDS: SupportedChainIds = {
+  sell: PortalsSupportedChainIds as unknown as ChainId[],
+  buy: PortalsSupportedChainIds as unknown as ChainId[],
```

Comment:
That's super weird, I think we could make `KnownChainIds` be a mapping of `Record<string, ChainId>` for sure to avoid those constants, anyway, not related to the PR and it would require too much changes, I'm happy with it